nextcloudappstore/core/templates/app/base.html

```diff
@@ -13,7 +13,7 @@
             </li>
             {% for cat in categories %}
             <li class="{% if cat.id == current_category.id %}active{% endif %}">
-                <a href="{% url 'category_list_category' cat.id %}">{{cat.name }}</a></li>
+                <a href="{% url 'app-list' cat.id %}">{{ cat.name }}</a></li>
```

Comment:
I find app-list really confusing here. I would at least use app-list-by-category. I'd guess we'll at least get an app-list-by-developer route at some point.